doc/how_to_guides/usage.md

```diff
@@ -31,66 +32,82 @@ Runtime options:
 
 #### Examples
 ```bash
-bundle exec rails generate onesie:task MyTask      # creates a normal priority task
-bundle exec rails generate onesie:task MyTask high # creates a high priority task
+bundle exec rails generate onesie:task MyTask      # creates a normal priority Task
+bundle exec rails generate onesie:task MyTask high # creates a high priority Task
+
 ```
 
 ### Rake
+The following commands should occur within a V2 shell
+(`dcc shell benchprep-v2`). For more information on setting up
+containers, please consult our
+[container setup guide](../infrastructure/how_to_guides/dev_containers/setup_guide.md).
```

Comment:
We should not include BenchPrep-specific infrastructure tooling in a gem's documentation